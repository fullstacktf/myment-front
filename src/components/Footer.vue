<template>
    <div class="footerContainer"> {{copyright}} {{author}}  {{year}}</div>
</template>

<script>
export default {
  data(){
      return {
      copyright: "derechos de copyright todos tochos ",
      author: "los seres todopoderosos a los que vosotros, simples usuarios, deb√©is reverencia",
      year: 2019
      }
  }
}
</script>

<style scoped>
.footerContainer{
    background: black;
    color: white;
    width: 100%;
    height: 100%;
    font-size: 12px;
    text-align: center;
    }

</style>
