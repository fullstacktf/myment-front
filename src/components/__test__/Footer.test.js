import Footer from '../Footer.vue';
import { shallowMount, createLocalVue } from '@vue/test-utils';
import Buefy from 'buefy';
import mock from '../../test/mockBack';

describe('Footer', () => {
  let localVue;
  beforeEach(() => {
    const Vue = createLocalVue();
    Vue.use(Buefy);
    localVue = Vue;
  });
  test('render component', () => {
    const wrapper = shallowMount(Footer, {
      localVue,
    });
    expect(wrapper.html()).toMatchSnapshot();
  });

  test('Check props', () => {
    
  });

  test('check Contents props', () => {
    
  });
});


