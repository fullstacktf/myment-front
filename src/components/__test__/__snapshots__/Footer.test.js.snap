// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer render component 1`] = `"<div class=\\"footerContainer\\"> derechos de copyright todos tochos  los seres todopoderosos a los que vosotros, simples usuarios, deb√©is reverencia  2019</div>"`;
